CONTRIBUTING.md

```diff
@@ -364,7 +361,7 @@ To inhibit the automatic addition of ToC, add the following to the front-matter
 `toc: false`
 
 This ToC generator depends on headings being correctly coded in the markdown
-portion of the doc site files. If a page has an incorrectly-formatted ToC, be
+the portion of the doc site files. If a page has an incorrectly-formatted ToC, be
```

Comment:
This line is part of the sentence above. The term is "the markdown portion of the doc site," so "the markdown the portion..." would not make sense. Please revert.